operator/pkg/apis/core/v1beta1/configconnector_types.go

```diff
@@ -66,6 +66,12 @@ type ConfigConnectorSpec struct {
 	//+kubebuilder:validation:Enum=Absent;Merge
 	//+kubebuilder:validation:Optional
 	StateIntoSpec *StateIntoSpecValue `json:"stateIntoSpec,omitempty"`
+
+	// Experiments are a list of strings that allow opting-in to previews/experiments.
+	// This functionality is generally not supported and not expected to work going forwards,
+	// the intention is to allow new functionality to be delivered to non-production environments quickly,
+	// proving the functionality before it is baked into the supported API.
+	Experiments []string `json:"experiments,omitempty"`
```

Comment:
Nice!

Reply:
Thanks, I think it's a good way to ship things quickly before baking them into fields, but would welcome other opinions/thoughts!  cc @cheftako 

(This branch also relies on a kubebuilder-declarative-pattern upgrade to get the latest tests, so also relies on controller-runtime being upgraded, hence I'm annoying @cheftako about updating controller-runtime!)